javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,253 @@
+//question 1:Item array removal------
```

Comment:
remove unrelated file from this PR.

Reply:
how to remove this? thanks for telling.

Reply:
Hi aina, how can I remove this?

Reply:
Chunmei, try to ask Ida (aina) on slack instead or just ask in your class channel, to get help with your question